aspnetcore-features/CheckboxListSolution/CheckboxList/CheckboxList.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Should be targeting .NET 7.

Reply:
Updated my VS to the latest version and .NET 7 is installed. Updated the project too to point to .NET 7.